bin/tee

```diff
@@ -37,7 +37,7 @@ unless ($nostdout) {
 $| = 1 if $unbuffer;
 
 for (@ARGV) {
-    if (!open($fh, (/^[^>|]/ && $mode) . $_)) {
+    if (!open($fh, '/^[^>|]/', ($mode) . $_)) {
```

Comment:
This one is suspect. Maybe leave this one alone.

Not that the stuff in parens is trying to determine the mode then append that to `$_`. The mode is not the pattern `/^[^>|]/`.